tests/kernel/smp/src/main.c

```diff
@@ -651,15 +651,11 @@ void ztest_post_fatal_error_hook(unsigned int reason, const z_arch_esf_t *pEsf)
 
 void entry_oops(void *p1, void *p2, void *p3)
 {
-	unsigned int key;
-
 	ztest_set_fault_valid(true);
 
-	key = irq_lock();
 	k_oops();
 	TC_ERROR("SHOULD NEVER SEE THIS\n");
 	rv = TC_FAIL;
-	irq_unlock(key);
 }
```

Comment:
Hi, @andyross , for the line 651~663, I have already removed them base on your suggestion, so it conflicts here. Sorry for the inconvenience.